gratipay/billing/payday.py

```diff
@@ -27,7 +27,7 @@
 
 
 with open('sql/payday.sql') as f:
-    PAYDAY = f.read()
+    PAYDAY = f.read() % {'minimum_charge': MINIMUM_CHARGE}
```

Comment:
How about the approach in 249379279a931a3ece8d29e2e9ec255665c997ad, @rohitpaulk?